ee/tabby-webserver/src/handler.rs

```diff
@@ -22,6 +25,7 @@ pub async fn attach_webserver(
     ui: Router,
     logger: Arc<dyn RawEventLogger>,
     code: Arc<dyn CodeSearch>,
+    conf: Arc<Config>,
```

Comment:
avoid use abbr for short words, just use config
```suggestion
    config: &Config,
```